### DISCUSSION TITLE
How to achieve the animation on  tailwindcss oficial website

### DISCUSSION THREAD

**Discussion by xhxhxhxh** [tailwindlabs/tailwindcss#3710] [Help] [answered]
I know the code effects is based on highlight.js, but how to achieve this animation effect.

**Comment by xhxhxhxh** [comment]
Maybe I found it, [glorious-demo](https://github.com/glorious-codes/glorious-demo)

**Comment by simonswiss** [comment, marked as answer]
Hey!

It's using [Framer Motion](https://www.framer.com/motion/).

The code fro the Tailwind CSS official docs website is completely open source. You'll find the code for the homepage animated sections in the `components/home` directory:

https://github.com/tailwindlabs/tailwindcss.com/tree/master/src/components/home

There's a lot of cool stuff happening in there

**Reply by xhxhxhxh** [reply]
Thanks！

**Comment by Lindsey182** [comment]
Thanks a ton for sharing your knowledge!

[Tell The Bell](https://tellthebell.site/)
